src/MongoDB.Driver.Core/MongoNodeIsRecoveringException.cs

```diff
@@ -74,29 +69,6 @@ public MongoNodeIsRecoveringException(ConnectionId connectionId, BsonDocument re
         protected MongoNodeIsRecoveringException(SerializationInfo info, StreamingContext context)
             : base(info, context)
         {
-            _result = (BsonDocument)info.GetValue("_result", typeof(BsonDocument));
```

Comment:
This line must be removed, but the constructor must be kept even though the body is empty.